content/uk/docs/tasks/tools/install-minikube.md

```diff
@@ -9,6 +9,10 @@ card:
 
 <!-- overview -->
 
+<!--This page shows you how to install [Minikube](/docs/tutorials/hello-minikube),
+a tool that runs a single-node Kubernetes cluster in a virtual machine on your personal computer.
+-->
+
 Ця сторінка описує як встановити [Minikube](/docs/tutorials/hello-minikube), інструмент який дозволяє
 запустити  Kubernetes кластер з однієї ноди у віртуальній машині на персональному комп'ютері.
```

Comment:
just nitpicking, it looks great but I would leave "your" in the translation. anyway, great work!